rpc/nodeagent/network/logicalnetwork/moc_nodeagent_logicalnetwork.proto

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache v2.0 license.
+
+syntax = "proto3";
+option go_package = "github.com/microsoft/moc/rpc/nodeagent/network";
+package moc.nodeagent.network;
+
+import "google/protobuf/wrappers.proto";
+import "google/protobuf/empty.proto";
+import "moc_common_common.proto";
+import "moc_common_networkcommon.proto";
+import "moc_common_notification.proto";
+
+message LogicalNetworkRequest {
+	repeated LogicalNetwork LogicalNetworks = 1;
+	Operation OperationType = 2;
+}
+
+message LogicalNetworkResponse {
+	repeated LogicalNetwork LogicalNetworks = 1;
+	google.protobuf.BoolValue Result = 2;
+	string  Error = 3;
+}
+
+message LogicalNetwork {
+	string name = 1;
+	string id = 2;
+	repeated LogicalNetworkIpam ipams = 3;
+	Status status = 4;
+	Entity entity = 5;
+	Tags tags = 6;
+}
```

Comment:
Any reason `NetworkType` is removed from here?

Reply:
@msftpgayam I was told it is not needed for Logical Network.

Reply:
@raghavendra-nataraj we will probably need this for the non-sdn scenario. Do we want to bring it in later?